### PR TITLE
Upgrade to elm 0.19

### PR DESCRIPTION
**Pull request by alex-tan**

### CODE REVIEW

File: src/Array/Extra.elm [hunk 1]
```diff
@@ -1,110 +1,124 @@
...
 
+
 # Slicing / resizing
+
 @docs resizelRepeat, resizerRepeat, resizelIndexed, resizerIndexed, splitAt, removeAt
 
+
 # Unsafe
+
 @docs getUnsafe
```
Comment (prikhi) [inline src/Array/Extra.elm]:
```suggestion
```

File: src/Array/Extra.elm [hunk 2]
```diff
@@ -1,110 +1,124 @@
...
 @docs zip, zip3, zip4, zip5
 
+
 # Slicing / resizing
+
 @docs resizelRepeat, resizerRepeat, resizelIndexed, resizerIndexed, splitAt, removeAt
 
+
 # Unsafe
+
```
Comment (prikhi) [inline src/Array/Extra.elm]:
```suggestion
```

File: src/Array/Extra.elm [hunk 3]
```diff
@@ -1,110 +1,124 @@
...
 @docs zip, zip3, zip4, zip5
 
+
 # Slicing / resizing
+
 @docs resizelRepeat, resizerRepeat, resizelIndexed, resizerIndexed, splitAt, removeAt
 
+
 # Unsafe
```
Comment (prikhi) [inline src/Array/Extra.elm]:
```suggestion
```

File: src/Array/Extra.elm [hunk 4]
```diff
@@ -1,110 +1,124 @@
...
 # Zips
+
 @docs zip, zip3, zip4, zip5
 
+
 # Slicing / resizing
+
 @docs resizelRepeat, resizerRepeat, resizelIndexed, resizerIndexed, splitAt, removeAt
 
+
```
Comment (prikhi) [inline src/Array/Extra.elm]:
```suggestion
```

File: src/Array/Extra.elm [hunk 5]
```diff
@@ -167,25 +190,13 @@ removeWhen pred xs =
 -}
 zip : Array a -> Array b -> Array ( a, b )
 zip =
-    map2 (,)
+    map2 (\a b -> ( a, b ))
```
Comment (prikhi) [inline src/Array/Extra.elm]:
```suggestion
    map2 Tuple.pair
```

**Comment by alex-tan** [comment]
👍 Thanks for reviewing. Should be good to go

**Comment by prikhi** [comment]
@jonboiser Ping. Do you want to review/merge this?

**Comment by prikhi** [comment]
@jonboiser Thanks, do you want to publish a new version? Or I can do it if you'd like.

**Comment by jonboiser** [comment]
@prikhi Could you go ahead and publish this? Thanks

**Comment by prikhi** [comment]
Yup, just pushed it.

**Comment by ccapndave** [comment]
Thanks to everyone for releasing the updated version!

**Pull request merged**